web/Services/PropertyService.cs

```diff
@@ -165,7 +170,14 @@ public bool IsUserAdminOrCorrectLandlord(BricksAndHeartsUser currentUser, int pr
         return propertyLandlordId == userLandlordId;
     }
 
-    public async Task<(List<PropertyDbModel> PropertyList, int Count)> GetPropertyList(string? sortBy, string? target, int page, int propPerPage)
+    public LandlordDbModel GetPropertyOwner(int propertyId)
+    {
+        return _dbContext.Properties.Include(p => p.Landlord).ThenInclude(l => l.Postcode)

```

Comment:
So I meant do 
```suggestion
        return _dbContext.Properties.Include(p => p.Landlord).Include(p => p.Postcode)

```

The Landlord postcode is still just a string as currently we aren't doing anything with it

Reply:
Ah I see! This method only returns the Landlord though, is there a reason to include the property's postcode as well?

Reply:
Yep you're totally right, ignore me